schemas/groups/notifications.json

```diff
@@ -5,37 +5,32 @@
   "definitions": {
     "notification": {
       "type": "object",
-      "properties": {
-        "method": {
-          "description": "Method to use to raise notifications",
-          "type": "array",
-          "items": {
-            "$ref": "../definitions.json#/definitions/alarmMethodEnum"
+      "allOf": [{
+        "$ref": "../definitions.json#/definitions/commonValueFields"
+      }, {
+        "properties": {
+          "method": {
+            "description": "Method to use to raise notifications",
+            "type": "array",
+            "items": {
+              "$ref": "../definitions.json#/definitions/alarmMethodEnum"
+            }
+          },
+          "state": {
+            "description": "Current notification state",
+            "$ref": "../definitions.json#/definitions/alarmState"
+          },
+          "message": {
+            "description": "Message to display or speak",
+            "type": "string"
           }
-        },
-        "state": {
-          "description": "Current notification state",
-          "$ref": "../definitions.json#/definitions/alarmState"
-        },
-        "message": {
-          "description": "Message to display or speak",
-          "type": "string"
-        },
-        "timestamp": {
-          "description": "Timestamp of the last update to this data",
-          "$ref": "../definitions.json#/definitions/timestamp"
-        },
-        "$source": {
-          "description": "Source of this data",
-          "$ref": "../definitions.json#/definitions/sourceRef"
         }
-      },
+      }],
       "required": [
         "method",
         "state",
         "message"
-      ],
-      "additionalProperties": false
```

Comment:
Sorry about sending you and myself on a wild goose chase: `*Of` and `additionalProperties: false` do not mix, this is (apparently!) a known problem with json schema. 

Instead the validator is configured at top level as no additional properties allowed, so you don't need to do that inside the schema files.